core/schema/src/main/resources/org/hawkular/schema/updates/schema-0.19.0.groovy

```diff
@@ -24,4 +24,70 @@ schemaChange {
   cql """
 ALTER TABLE data WITH COMPRESSION = {'sstable_compression': 'DeflateCompressor'};
 """
-}
\ No newline at end of file
+}
+
+schemaChange {
+  version '3.1'
+  author 'jsanda'
+  tags '0.19.x'
+  cql """
+CREATE TABLE rollup60 (
+    tenant_id text,
+    metric text,
+    shard  bigint,
+    time timestamp,
+    min double,
```

Comment:
I forgot to include some relevant details when I opened the PR. I was going back and forth about whether or not to use typed columns versus storing everything as a blob for the very reasons you mention. It mean more encoding/decoding on the client side, but it gives us more flexibility and maybe even more efficient storage. I am not sure that a CQL map gives us that same flexibility.

I have also considered the same things about 1 table versus many. Using one table allows us to introduce different rollups without requiring schema changes. If we go that route, we also need to create Infinispan caches on demand. Creating the different rollups and caches is easy enough, but keeping track of everything could get cumbersome.  I do question how much of that flexibility we will need in practice.